src/components/10-atoms/fieldset/story.js

```diff
@@ -17,10 +17,15 @@ storiesOf('Components|Atoms/Fieldset', module)
   .add('Fieldset', () => {
     const error = text('error', '');
     const horizontal = boolean('horizontal', false);
+    const disableresponsivestretch = boolean('disableResponsiveStretch', false);
```

Comment:
This knob do not have any effect at this story. Try to show that knob in action (perhaps with some hints for the user like "resize your browser to a mobile width") or create a fieldset demo.

Reply:
(outdated)